### PR TITLE
Add auto scaling docs

### PR DESCRIPTION
**Pull request by aayushmahapatra**
Only description with coming soon.

### CODE REVIEW

File: pages/compute/scaling/no.mdx [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import { Callout } from "nextra-theme-docs";
+import Navigation from "components/navigation";
+
+# No Scaling
```
Comment (rekpero) [inline pages/compute/scaling/no.mdx:4]: why is this page here? we have no-scaling page in the docs right? I think something went wrong with the merge conflict. please check @aayushmahapatra

**Review by rekpero — CHANGES_REQUESTED** [review]

File: pages/compute/scaling/manual-scaling.mdx [hunk 2]
```diff
@@ -8,6 +8,16 @@ to increase or decrease the allocated resources, such as CPU, memory, or storage
 This type of scaling provides users with direct control over resource adjustments, allowing for optimization based
 on specific workload characteristics and immediate operational needs.
 
+<Callout type="warning">
+  The Manual Scaling feature is currently in its closed alpha stage. Potential
+  irregularities may lead to brief service downtimes. We recommend utilizing
+  this feature exclusively in testing or development environments until its
+  official release for optimal stability. If you're interested in obtaining
+  alpha access, please use this
+  [https://meetings.hubspot.com/prashant-maurya/scaling-demo](https://meetings.hubspot.com/prashant-maurya/scaling-demo)
```
Comment (rekpero) [inline pages/compute/scaling/manual-scaling.mdx:17]:
```suggestion
  [link](https://meetings.hubspot.com/prashant-maurya/scaling-demo)
```

**Review by rekpero — CHANGES_REQUESTED** [review]
Check my comment

File: pages/compute/scaling/manual-scaling.mdx (hunk 2 above)
Comment (aayushmahapatra) [reply pages/compute/scaling/manual-scaling.mdx:17]: oh 😂

**Review by rekpero — APPROVED** [review]

**Pull request merged**